### PR TITLE
Upsert response

### PR DESCRIPTION
**Pull request by mrgaaron**

### CODE REVIEW

File: TempoIQ/Models/UpsertResponse.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        public UpsertResponse(IDictionary<String, DeviceStatus> data) : base()
+        {
+            foreach(var pair in data)
+                this.Add(pair.Key, pair.Value);
+        }
+         
+        /*public UpsertResponse(Dictionary<String, DeviceStatus> statuses
+        {
+            if (statuses == null)
+                throw new BadImageFormatException("YOU SUCK");
```
Comment (selecsosi) [inline TempoIQ/Models/UpsertResponse.cs]: Should probably not leave this in there

File: TempoIQ/Models/UpsertResponse.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            this.statuses = statuses;
+        }*/
+
+        public bool Success
+        {
+            get
+            {
+                foreach (DeviceStatus status in this.Values)
```
Comment (jcc333) [inline TempoIQ/Models/UpsertResponse.cs]: If you ever have to do something like this in the future, you might like to know about: https://msdn.microsoft.com/en-us/library/vstudio/bb548541%28v=vs.100%29.aspx

File: TempoIQ/Utilities/Executor.cs [hunk 3]
```diff
@@ -61,7 +61,7 @@ public Result<T> Execute<T>(Method method, string resource, object body, string
             request.AddBody(body);
             if (mediaTypeVersions.Any())
                 request.AddHeader("Accept", String.Join(",", mediaTypeVersions));
-            if (contentType != null && contentType != "")
+            if (!String.IsNullOrEmpty(contentType))
```
Comment (jcc333) [inline TempoIQ/Utilities/Executor.cs]: +1 good-developer-points

**Comment by jcc333** [comment]
:+1:

**Pull request merged**